.npmignore

```diff
@@ -0,0 +1,9 @@
+node_modules/
+images/
+packages/autocannon-ui-frontend/
+packages/autocannon-ui-backend/.test_results/
+packages/autocannon-ui-backend/test/
+packages/autocannon-ui-backend/test/
+packages/autocannon-ui-backend/.eslintrc
+packages/autocannon-ui-backend/.prettierrc
+packages/autocannon-ui-backend/.taprc
```

Comment:
let's try to avoid such specific exclude rules, they're hard to maintain. surely there are simple rules we can have which are still a good compromise. remember this is not a production package, a few kb more won't hurt anybody

Reply:
Yepp. Will simplify